### PR TITLE
misc: Remove members.json

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
As Eleventy makes the whole directory available in a map structure this change removes the members.json file. Adding things twice makes it error prone, and it gives us no additional benefit.

### CODE REVIEW

File: src/team.njk [hunk 1]
```diff
@@ -5,53 +5,57 @@ layout: layouts/nohero.njk
...
+            {% set member = team[name] %}
+
             <div class="team-headshot w-full max-w-xs m-auto">
-                {% set imageSrc = ["./images/team/headshot-", team[member].headshot ] | join %}
-                {% set imageAlt = ["Photo of ", team[member].name ] | join %}
+                {% set imageSrc = ["./images/team/headshot-", member.headshot ] | join %}
+                {% set imageAlt = ["Photo of ", member.name ] | join %}
+		{{ imageSrc | log }}
+		{{ imageAlt | log }}
                 {% image imageSrc, imageAlt, [256] %}
```
Comment (ZJvandeWeg) [inline src/team.njk:16]: @Pezmc This line doesn't work for me now anymore, and I don't understand the why. Do you have an idea?

Comment (Pezmc) [reply src/team.njk:12]: I tweaked the loop in https://github.com/flowforge/website/pull/520/commits/3b32b4f8838be87ea3a92a9c1529af0f7312dea7, which seems to have fixed your image rendering

**Comment by joepavitt** [comment]
@ZJvandeWeg just noticed a consequence of this - our /team page is now ordered alphabetically, not by seniority/join date: https://flowforge.com/team/

I vote we put this back

**Comment by Pezmc** [comment]
@joepavitt Should we add a "sort/rank" value to the JSON? Then order by that desc?

**Comment by joepavitt** [comment]
Yeah that's sensible

**Comment by joepavitt** [comment]
Although better going ascending, as it grows better

**Pull request merged**